docs/Users_Guide/lulcncarhpc.rst

```diff
@@ -0,0 +1,170 @@
+
+NCAR HPC systems provide the computational resources needed for efficient METplus processing of large I-WRF datasets. This exercise uses containerized METplus applications, which simplifies the setup work needed to run the verification and ensures consistency across different HPC environments.
+
+It is recommended that you follow the instructions in each section in the order presented to avoid encountering issues during the process. This guide assumes that I-WRF LULC simulations have already been completed and focuses specifically on the METplus verification workflow.
+
+
+Prepare to Use NCAR HPCs
+========================
+
+To get started with running METplus verification on NCAR HPC systems, you will need access to one of the supported platforms (Derecho, Casper, or other NCAR computing resources). If you do not already have access, you will need to request an account through the NCAR Computing and Information Systems Laboratory (CISL).
+
+NCAR HPC access is typically provided to researchers affiliated with NCAR, university collaborators, or those with approved allocations. If you need to request access:
+
+* Visit the NCAR CISL user registration page to create an account
+* Provide information about your research project and institutional affiliation
+* Wait for account approval, which typically takes 1-2 business days
+* Once approved, you can log in to the HPC systems using SSH
+
+Most NCAR HPC systems use a batch job scheduling system for computational work. While this exercise can be run interactively for demonstration purposes, production runs should be submitted through the job scheduler. Refer to the NCAR HPC documentation for specific guidance on your target system.
+Once you have access to an NCAR HPC system, you can log in using SSH from your local machine or through NCAR's web-based interfaces where available.
+
+Preparing the Environment
+=========================
+
+With your NCAR HPC account active and you logged in to the system, you can now set up the environment and create the necessary directories to run METplus verification. You will only need to perform these steps once per system, as the directory structure and configuration will remain available for future use.
+
+The following sections instruct you to issue numerous Linux commands in your shell. The commands in each section can be copied and pasted into your shell terminal.
+
+Load Required Modules
+---------------------
+
+NCAR HPC systems use environment modules to manage software. Load the Apptainer module which provides the containerization software needed to run METplus::
+
+   module load apptainer
+
```

Comment:
Surprisingly, running `module load` in tcsh prior to switching to `bash` seems to work. `apptainer` is still in my path after switching shells.
```suggestion
Confirm Bash Shell
------------------

The following commands are formatted to run in the Linux Bash shell, which you may or may not be running by default. Copy and paste the following command to print your default shell::

   echo $SHELL

If needed, switch to the :code:`bash` shell by running::

   bash

Note that you can modify your default login shell through the `Systems Account Manager (SAM) <https://sam.ucar.edu/>`_ but changes can take up to several hours to take effect.
```